--- a/cmake/instrument-script-server-config.cmake.in
+++ b/cmake/instrument-script-server-config.cmake.in
@@ -5,7 +5,7 @@ include(CMakeFindDependencyMacro)
 find_dependency(Threads REQUIRED)
 find_dependency(Lua REQUIRED)
 find_dependency(nlohmann_json CONFIG REQUIRED)
-find_dependency(grpc CONFIG REQUIRED)
+find_dependency(gRPC CONFIG REQUIRED)
 find_dependency(yaml-cpp CONFIG REQUIRED)
 find_dependency(sol2 CONFIG REQUIRED)
 find_dependency(cxxopts CONFIG REQUIRED)
